docs/csharp/misc/cs1626.md

```diff
@@ -10,7 +10,7 @@ ms.assetid: 3ba03383-eb24-4fd8-bf40-8b0f7d6baf0d
 # Compiler Error CS1626
 Cannot yield a value in the body of a try block with a catch clause  
   
- A yield statement is not allowed in a try block if there is a catch clause associated with the try block. To avoid this error, move the yield statement out of the catch clause.  
+ A yield statement is not allowed in a try block if there is a catch clause associated with the try block. To avoid this error, either move the yield statement out of the try block or remove the catch block. 
```

Comment:
The question here is where the yield statement can be moved. It can't be moved to the catch block (generates CS1631: Cannot yield a value in the body of a catch clause) or to the finally block (generates CS1625: Cannot yield in the body of a finally clause). I'd rephrase this as: "move the yield statement out of the try/catch/finally block, or remove the catch block.

Reply:
Yes, that is much clearer.  Thanks for reviewing and suggesting, I will make the change.

Reply:
@rpetrusha I have completed the change.  Please take a look when you have time.